spec/system/distribution_system_spec.rb

```diff
@@ -37,7 +40,9 @@
 
         select @partner.name, from: "Partner"
         expect do
-          click_button "Save"
+          accept_alert do
+            click_button "Save"
+          end
         end.not_to change { ActionMailer::Base.deliveries.count }
```

Comment:
For instance, this would impact here, think that the message can still be delivered after a few seconds but the matcher will pass right away.

Reply:
Exactly. I'll improve that. Thanks again!